tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -300,14 +300,29 @@ object TastyFormat {
    * is able to read final TASTy documents if the file's
    * `MinorVersion` is strictly less than the current value.
    */
-  final val ExperimentalVersion: Int = 3
+  final val ExperimentalVersion: Int = 0
```

Comment:
This is fine for the release but we shouldn't publish a nightly with experimental features enabled but a non-experimental version, so we need an extra commit after this one which won't be backported to bump MinorVersion and make ExperimentalVersion non-zero (... and now I'm realizing things would be easier if we didn't need to change two fields to switch between experimental and non-experimental, oh well we can always change the exact meaning of ExperimentalVersion in the future if we want).

Reply:
should this PR bump the versions to the next minor + experimental, and then the backport does the present version change?

Reply:
Ok right I see what you mean now, both commits in this PR

Reply:
so in this case the minor version is being increased even though there are no changes to tasty, in this way we could not set a nightly to be the `baseVersion` of a stable release, because it would be experimental